.travis.yml

```diff
@@ -0,0 +1,47 @@
+language: cpp
+dist: bionic
+git:
+  submodules: false
+  
+matrix:
+  include:
+  - os: linux
```

Comment:
Why only linux?

Reply:
Please, read the comment above. We're planing to support not only linux. Converting pull to draft at the moment.